python/local_program/aws-py-voting-app/__main__.py

```diff
@@ -95,10 +95,7 @@
 
 # Attaching execution permissions to the task role
 task_policy_attachment = aws.iam.RolePolicyAttachment("app-access-policy", role=app_task_role.name,
-                                                      policy_arn="arn:aws:iam::aws:policy/AmazonEC2ContainerServiceFullAccess")
-
-lambda_policy_attachment = aws.iam.RolePolicyAttachment("app-lambda-policy", role=app_task_role.name,
-                                                        policy_arn="arn:aws:iam::aws:policy/AWSLambdaFullAccess")
```

Comment:
The Lambda permission is not actually needed for this example (tested and verified manually in https://github.com/pulumi/examples/pull/894)